packages/hebao_v1/contracts/iface/Controller.sol

```diff
@@ -37,6 +37,7 @@ contract Controller
     // The address to which all colletable tokens/ether in modules will
     // be sent to.
     address public collectTo;
+    uint    public lockPeriod;
```

Comment:
I think this will be module depend, but it's not like the module is forced to use this value so I guess it can't hurt to have a default value.

Reply:
that's right.  I changed it to `defaultLockPeriod` and add another `lockWallet` function with a second parameter called `lockPeriod`